{
  "submission_id": "S78",
  "user_id": "U1",
  "problem_id": "P6",
  "timestamp": "2025-09-27T08:00:22Z",
  "status": "Running... (Testcase 1)",
  "language": "python",
  "test_results": [
    {
      "test_case_number": 1,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.01,
      "memory_usage": 7.828125,
      "actual_output": "",
      "expected_output": "EVEN\nODD\nEVEN\nEVEN\nODD",
      "input": "5\n0\n1\n2\n1000000000000000000\n999999999999999999\n"
    }
  ]
}